src/components/molecules/ZopaFooter/__snapshots__/ZopaFooter.test.tsx.snap

```diff
@@ -82,7 +82,7 @@ and entered on the Financial Services Register under firm registration number 71
       <p
         class="c3"
       >
-        © Zopa Limited 2018 All rights reserved. 'Zopa' and the Zopa logo are trade marks of Zopa Limited. 
+        © Zopa Limited 2019 All rights reserved. 'Zopa' and the Zopa logo are trade marks of Zopa Limited. 
```

Comment:
If this year is dynamic we will need to update the snapshot manually every year. Unless we mock the date which I think it's the best approach in this case. If you need help let me know :)

Reply:
@poteirard agree, and done!